modules/permissions/templates/jenkins_lambda_deploy_policy.json.tpl

```diff
@@ -0,0 +1,19 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Action": [
+        "lambda:UpdateFunctionCode",
+        "lambda:PublishVersion",
+        "s3:PutObject",
+        "s3:GetObject"
+      ],
+      "Resource": [
+        "arn:aws:lambda:eu-west-2:${account_id}:function:tdr-ecr-scan-notifications-mgmt",
+        "arn:aws:s3:::tdr-backend-code-mgmt/*"
+      ]
+    }
+  ]
+}
```

Comment:
Missing line break

Reply:
I've added this one and the one in the variables file.